### ISSUE TITLE
New feature request: Script and stored proc debugging

### ISSUE THREAD

**Issue by doitlikepruett** [Microsoft/sqlopsstudio#690]
It would super great if there was a built in debugger like SSMS has.

**Comment by kburtram** [comment]
Currently the SQL Server debugging protocol is built on DCOM and Windows RPC, so we don't have a cross-platform stack available.  I'll add this to the backlog, perhaps this is something we could look at as a Windows-only feature based on customer demand.  Building a new debugging protocol can't be done at the tools level and would require significant engine work (but it has been discussed internally).

**Comment by TylerLeonhardt** [comment]
👍 would also like Azure Data Studio to support debugging so the PowerShell extension's debugger can be used!

**Comment by tgchip** [comment]
Would an Azure Data Studio debugger capable of debugging Postgres stored procedures be considered at some point?

**Comment by svanschuyver** [comment]
I would really love to see this feature.

**Comment by pvspain** [comment]
Yes! Yes! Please add a debugger. This would be a massive timesaver
